pkg/process/process.go

```diff
@@ -137,15 +138,26 @@ func (p *Process) Stop() {
 }
 
 func (p *Process) StopWithSignal(signal syscall.Signal) {
-	needStop := false
+	needStop, needTimeoutKill := false, false
+	now := time.Now()
+
 	p.lock.Lock()
 	if p.State != StateStopping && p.State != StateStopped && p.State != StateError {
 		p.State = StateStopping
+		p.DeletionTimestamp = &now
 		needStop = true
 	}
+	// Retry the deletion if the process is not stopped in 30 seconds
```

Comment:
60 seconds?

WaitInterval is 100ms and WaitCount is 600.

Reply:
Yeah. I forgot to update this comment. Will do it with a new PR